engine/battle/core.asm

```diff
@@ -2648,8 +2653,9 @@ SelectMenuItem:
 	jr z, .disabled
 	ld a, [wPlayerBattleStatus3]
 	bit 3, a ; transformed
-	jr nz, .dummy ; game freak derp
-.dummy
+	jr nz, .transformedMoveSelected
+.transformedMoveSelected
+	; Allow moves copied by Transform to be used.
```

Comment:
What line(s) is this comment associated with? The game has a check for whether a move was selected while transformed, which is pointless since it falls through to the same code regardless. A comment here could briefly explain what happens when a move is selected while transformed to clarify why the check was there and whether it can be removed or if there's a bug that needs fixing.

Reply:
The comment is just describing the subroutine's purpose in general. Removing it doesn't break Transform itself, but it prevents the transformed Pokémon from using the moves it copied from the target.